src/others/helpers/map-utils.ts

```diff
@@ -36,8 +36,23 @@ export const mapAidRequestsToFeatures = (decodedAidRequestGroupedByLocation: Dec
                 addAggregatedFeaturePerLocationAndDate(locationGroup, locationAndDateRequests, features);
             })
         });
-    })
-    return features;
+    });
+    var maxPerCategory: { [id:string]: number;} = {};
+    features.forEach(function (feature) {
+      if (feature.properties) {
+        if (!(feature.properties.category in maxPerCategory)) {
+        	 maxPerCategory[feature.properties.category] = 0;
+        }
+        maxPerCategory[feature.properties.category] = Math.max(maxPerCategory[feature.properties.category], feature.properties.amount);
+      }
+    }, Object.create(null));
```

Comment:
Any particular reason for this? Not sure I follow

Reply:
Thanks, removed